jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/CodeWhispererCodeScanManager.kt

```diff
@@ -166,17 +171,21 @@ class CodeWhispererCodeScanManager(val project: Project) {
         var codeScanResponseContext = defaultCodeScanResponseContext()
         var getProjectSize: Deferred<Long?> = async { null }
         val connection = ToolkitConnectionManager.getInstance(project).activeConnectionForFeature(CodeWhispererConnection.getInstance())
+        var codeScanJobId: String? = null
+        var language: CodeWhispererProgrammingLanguage? = null
```

Comment:
nit: should we really need nullable here, tho I guess it's from the original PR (might miss to capture this previously)
I think `fun programmingLanguage()` will return a non-null value. If nullable doesn't add extra handling here that's fine.

Reply:
have changed to non-null for language.